chart/values.yaml

```diff
@@ -124,6 +124,17 @@ controlPlane:
     k8s:
       # Enabled specifies if the K8s distro should be enabled. Only one distro can be enabled at the same time.
       enabled: false
+      # Version specifies k8s components (scheduler, kube-controller-manager & apiserver) version.
+      # It is a shortcut for controlPlane.distro.k8s.apiServer.image.tag,
+      # controlPlane.distro.k8s.controllerManager.image.tag and
+      # controlPlane.distro.k8s.scheduler.image.tag
+      # If e.g. controlPlane.distro.k8s.version is set to v1.30.1 and
+      # controlPlane.distro.k8s.scheduler.image.tag
+      # (or controlPlane.distro.k8s.controllerManager.image.tag or controlPlane.distro.k8s.apiServer.image.tag)
+      # is set to v1.31.0,
+      # value from controlPlane.distro.k8s.<controlPlane-component>.image.tag will be used
+      # (where <controlPlane-component is apiServer, controllerManager and scheduler).
+      version: ""
```

Comment:
nit: Should we add a comment above the actual default tags below that if the tag is bumped the `$defaultTag` inside the _init-containers.tpl should also be bumped to the same value?!

Reply:
I can add a unit test in `config/config_test.go` that will check if `controlPlane.distro.<component>.image.tag` values are in sync with `$defaultTag` value in `_init-containers.tpl`. Then if anyone updates image tags without the defaultTag, this test will fail. WDYT?

Reply:
Sounds good, if it's not too much effort 🙂

Reply:
not at all, added :+1: